### ISSUE TITLE
Changing parameter group on aws_elasticache_replication_group crashes terraform

### ISSUE THREAD

**Issue by nara-tup** [hashicorp/terraform#9097]
- Create a elasticache replication group. This works.
- Add a parameter group. This works.
- Update elasticache replication group to change the parameter group name. Terraform crashes.

Here is a sample tf and crash log attached.

```
resource "aws_elasticache_parameter_group" "allkeys-lru" {
  name = "allkeys-lru"
  family = "redis2.8"

  parameter {
    name = "maxmemory-policy"
    value = "allkeys-lru"
  }

}

resource "aws_elasticache_replication_group" "cache-rep-group-1" {
  replication_group_id          = "cache-rep-group-1"
  replication_group_description = "cache-rep-group-1"
  engine                        = "redis"
  engine_version                = "2.8.24"
  node_type                     = "cache.m3.medium"
  number_cache_clusters         = 2
  port                          = 6379
  parameter_group_name          = "${aws_elasticache_parameter_group.allkeys-lru.name}"
  availability_zones            = ["us-east-1a", "us-east-1c"]
  subnet_group_name             = "prod"
  security_group_ids            = ["sg-xxxxxx"]
  automatic_failover_enabled    = true
  tags = {
    "env_name"        = "production"
    "service_name"    = "service1"
  }
}
```

[crash.log.zip](https://github.com/hashicorp/terraform/files/497029/crash.log.zip)

**Comment by stack72** [comment]
Hi @nara-tup 

Thanks for reporting this issue - I have found exactly what the problem is and have started the work to get this patched for terraform 0.7.5

Apologies if this is causing you problems right now

Paul

**Comment by nara-tup** [comment]
Awesome!
